src/main/kotlin/org/wycliffeassociates/otter/common/domain/content/RecordTake.kt

```diff
@@ -67,22 +67,34 @@ class RecordTake(
         val takeNumber = "%02d".format(number)
 
         // Compile the complete filename
-        return listOf(
-                languageSlug,
-                rcSlug,
-                "b$bookNumber",
-                bookSlug,
-                "c$chapterNumber",
-                "v$verseNumber",
-                "t$takeNumber"
-        ).joinToString("_", postfix = ".wav")
+
+        return if (chunk.labelKey == "chapter") {
```

Comment:
the complexity here of creating a file name has progressed to the point of necessitating a class